frontend/src/styling/global.ts

```diff
@@ -20,7 +20,7 @@ export const GlobalStyle = createGlobalStyle`
 
   /* Typography */
 
-  h1, h2, h3, h4, h5, h6, p, a, span, li, strong, label, input, textarea, option, select, button {
+  * {
```

Comment:
Is there a specific reason you use `*` instead of `body`?

Also, this change should happen in a separate pull request. Could you create a new pull request for this which only has this change? You can call it "Set default font for the application" or something similar.

Reply:
Yes. This is to reset the padding/margin of every html element. The reasoning behind this is to force the developer to implement their own styling for all elements, and not rely on the user-agent stylesheets, who have a tendency to differ slightly, leading to inconsistencies in padding/margin.

Reply:
Changing this to body would require us to manually remove the padding/margin on all other elements touched by the user-agent stylesheet.